src/w3c/headers.js

```diff
@@ -537,6 +537,13 @@ export function run(conf) {
       "If one of '`wg`', '`wgURI`', or '`wgPatentURI`' is an array, they all have to be."
     );
   }
+  if (conf.isCGBG && !conf.wg) {
+    pub(
+      "error",
+      "[`wg`](https://github.com/w3c/respec/wiki/wg)" +
+        " configuration option is required for CG documents."
```

Comment:
Nit: this applies to BG documents too, so better to say “required for this kind of document”.